poster/src/index.ts

```diff
@@ -38,7 +39,8 @@ async function run() {
   }
 
   try {
-    await main(argv.sources, argv.posterKey, argv.viewAddress, argv.viewFunction, web3);
+    web3.eth.transactionConfirmationBlocks = 10
```

Comment:
Just curious-- this is going to break dev right, since you only get one block confirmation on ganache?

Reply:
good point, did no mean for this to overwrite that dev change.